types/onoff.d.ts

```diff
@@ -0,0 +1,42 @@
+
+declare type Directions = "in" | "out";
+declare type Edges = "none" | "rising" | "falling" | "both";
+declare type GpioOptions = {
```

Comment:
I'm not actually familiar with TypeScript so I'm surprised to see that GpioOptions isn't used anywhere, is this correct?

Reply:
You are right, this is an error on my part. It defines the options used in the constructor which I forgot to add.